### DISCUSSION TITLE
Railway部署服务端后用一键部署客户端命令报错有没有同样的情况

### DISCUSSION THREAD

**Discussion by DifferentSparkler** [zdz/ServerStatus-Rust#50] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/38945550/171319030-6b47d022-a51a-48bb-a6e8-2781a4462d95.png)
curl -sSf "https://serverstatus-rust-production-ef61.up.railway.app/i?pass=pp&gid=g1&alias=$(hostname)" | bash
用的这个命令一键部署客户端报错

curl: (22) The requested URL returned error: 401

**Comment by zdz** [comment, marked as answer]
对一下组id和密码

**Comment by DifferentSparkler** [comment]


**Reply by DifferentSparkler** [reply]
我这个好像没有分组哦  密码都是admin

**Reply by zdz** [reply]
把分组配置加上吧

**Reply by DifferentSparkler** [reply]
感觉没必要分组一个人也不可能有很多服务器  有没有不用分组配置的命令默认添加到一个组

**Reply by zdz** [reply]
目前没有默认分组，分组是适配不同人不同的需求，例如 [#28](https://github.com/zdz/ServerStatus-Rust/discussions/28) ，分组可以让部署更简单一点，你自己加一个组来当默认分组也不麻烦，就加三行配置

**Comment by DifferentSparkler** [comment]


**Reply by zdz** [reply]
修改  gid 和 password 这两个字段即可

**Reply by DifferentSparkler** [reply]
我记得以前改了参数railway后自动重新部署这次好像没有重新部署

**Reply by zdz** [reply]
可以随便加一个变量 a =b 来触发一下

**Reply by DifferentSparkler** [reply]
我在试一下  谢谢你的耐心解答

**Comment by DifferentSparkler** [comment]
![image](https://user-images.githubusercontent.com/38945550/171323637-46a710be-216d-4102-bfbd-3a3cd40fc0d2.png)
终于解决困扰我好多天的问题了 自从部署好后就只添加了一个机器  现在只需要一个命令就可以添加客户端了 太好用了
